examples/ConnectFour.bgl

```diff
@@ -30,7 +32,6 @@ initialBoard!(x,y)= Empty
 goFirst : Player
 goFirst = A
 
--- built in next() is defined only for X and O
```

Comment:
False comment! There is not more built-in next

Reply:
Yup!